src/ipc/messages.capnp

```diff
@@ -0,0 +1,22 @@
+@0xa4478fe5ad6d80f5;
+
+using Cxx = import "/capnp/c++.capnp";
+$Cxx.namespace("ipc::messages");
+
+interface Node {
+    helpMessage @0 (mode :Int32) -> (value :Text);
+    handleInitMessage @1 (callback: InitMessageCallback) -> (handler :Handler);
+    wallet @2 () -> (wallet :Wallet);
+}
+
+interface Wallet {
+    getBalance @0 () -> (value :Int64);
```

Comment:
Pretty nice. So the IPC endpoint exposes multiple "objects". It can also expose multiple instances of one class? (e.g. for multiwallet?).